BritneyS/SFWCardsAgainstHumanity
# Card review

## What you did :question:
- Added and deleted some black card and white card phrases
- No app behavioral changes

## How you did it :white_check_mark:
- Deleted phrases by turning them into empty strings
- Added phrases in existing elements that had previous been empty strings

## How to test it :microscope:
- Run the app
- Tap 'Play!'
- White and black card phrases should appear in Play View as expected




## Screenshots (if applicable) :camera:
![simulator screen shot - iphone xr - 2018-11-01 at 10 13 23](https://user-images.githubusercontent.com/8409475/47857087-27bd2780-ddbf-11e8-817d-5175e505780f.png)
![simulator screen shot - iphone xr - 2018-11-01 at 10 13 31](https://user-images.githubusercontent.com/8409475/47857191-63f08800-ddbf-11e8-91f5-ee5c5f69f4fe.png)
